_posts/2018-01-25-Disobey.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Disobey 2018 was a blast!
+author: lokori
+excerpt: I visited [Disobey](https://disobey.fi/) for the second time a week ago. What a great experience again, in so many levels! What makes Disobey so great and why you might want to come there in 2019?
+tags:
+- disobey
+- hacking
+- infosec
+- software security
+- devsecops
+---
+
+## There and back again
+
+It felt very similar to [last year's Disobey](http://dev.solita.fi/2017/01/19/Disobey.html). This time I was a speaker and had some other duties as well, but I was still able to enjoy some very good talks and participate in the CTF competition. I left the web application hacking workshop to my colleagues (big thanks to Timo Hulkkonen especially) since I had to leave early due to personal reasons. I also missed the legendary after party, but I was more interested in learning and meeting people than partying and drinking anyway.
+
+## From hackers to hackers
+
+Disobey is organized as a non-profit conference, which keeps the ticket prizes low. The
+organizers are a mix of security experts and software professionals from different companies 
+as well as people who simply love hacking and hackers in general. The audience are their peers which makes
```

Comment:
people who simply love hacking and hackers in general -> love hacking and hackers? laittaisin pilkun tuohon and-sanaa ennen
there were roughly 800 people there -> there were roughlt 800 people present